cmd/bankapp/main.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+	"github.com/gorilla/mux"
+	"github.com/iliyanmotovski/bankv2/bank/api"
+	"github.com/iliyanmotovski/bankv2/bank/domain"
+	"github.com/iliyanmotovski/bankv2/bank/persistence"
+	"github.com/justinas/alice"
+	"gopkg.in/mgo.v2"
+	"net/http"
+	"time"
+)
+
+func main() {
+	session, err := mgo.DialWithTimeout("localhost:27016", time.Second*10)
+	if err != nil {
+		fmt.Println(err)
+	}
+	defer session.Close()
+	session.SetMode(mgo.Monotonic, true)
+
+	renew := domain.Renew{RenewSessionCounter: make(chan bool)}
+
+	r := mux.NewRouter()
+
+	mongoSessionRepo := persistence.NewSessionRepository(*session, "bank")
+
+	mongoImplementedUserStore := domain.UserStore(mongoSessionRepo)
+	mongoImplementedUserSessionStore := domain.SessionStore(mongoSessionRepo)
```

Comment:
Looks duplicated with the sessionRepository. I think that you have separated the same thing.